base/common/src/org/dogtagpki/nss/NSSExtensionGenerator.java

```diff
@@ -253,6 +254,16 @@ public AuthInfoAccessExtension createAIAExtension() throws Exception {
             extension.addAccessDescription(method, location);
         }
 
+        // Call AuthInfoAccessExtension.encode() to generate the extensionValue,
+        // otherwise the extensionValue will be null.
+        //
+        // TODO: Implement AuthInfoAccessExtension.getExtensionValue() to generate
+        // the extensionValue whenever it's needed.
+
+        try (ByteArrayOutputStream os = new ByteArrayOutputStream()) {
```

Comment:
I'm fine keeping this change, but perhaps it was meant to be part of a separate PR?

Reply:
Yeah, this is actually unrelated to ACME database, I just thought it's small enough to stick into this PR :)
The problem appears when I was testing the latest code. I'm not sure why it didn't appear before.